ledger/ledger-api-common/src/test/suite/scala/com/digitalasset/ledger/api/tls/TlsConfigurationTest.scala

```diff
@@ -46,6 +64,43 @@ class TlsConfigurationTest extends AnyWordSpec with Matchers with BeforeAndAfter
   }
 
   "TlsConfiguration" should {
+
+    "configure server with TLS protocol versions" which {
+      "is TLS 1.3" in {
```

Comment:
```suggestion
      "is 1.3" in {
```
We already use the TLS in this sentence once

Reply:
Ok